mindone/pipelines/text_classifiers/preprocess/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .corpus_cleaning_kit import en_cleaning, zh_cleaning
```

Comment:
`mindone.pipelines` only use in `examples/detect_chatgpt/detector_roberta.ipynb#L46`, move it to `examples/detect_chatgpt`?

Reply:
moved